tests/Security/Core/Authentication/Provider/OAuthProviderTest.php

```diff
@@ -221,14 +220,17 @@ public function testTokenRefreshesWhenExpired(bool $authenticated): void
             $serviceLocator
         );
 
-        $user = new User();
+        $user = $userKnown ? new User() : null;
 
         $userProviderMock = $this->getOAuthAwareUserProviderMock();
         $userProviderMock->expects($this->once())
             ->method('loadUserByOAuthUserResponse')
-            ->willReturn($user);
+            ->willReturn($user ?? $user = new User());
```

Comment:
Why not just `$user`? You have: `$user = $userKnown ? new User() : null;` at line 223.

Reply:
You are right that can be simplified.